pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/PdfSql.java

```diff
@@ -137,15 +137,19 @@ default int insertPhysicianInstitutionTemplate(PhysicianInstitutionTemplate temp
     @SqlUpdate("insert into pdf_activity_date_substitution (pdf_substitution_id, study_activity_id) values (:subId, :activityId)")
     int insertActivityDateSubstitution(@Bind("subId") long substitutionId, @Bind("activityId") long activityId);
 
-    @SqlUpdate("insert into pdf_answer_substitution (pdf_substitution_id, question_stable_code_id)"
-            + "(select :subId, question_stable_code_id"
-            + "   from question_stable_code as qsc"
-            + "  where qsc.stable_id = :stableId"
-            + "    and qsc.umbrella_study_id = (select study_id from study_activity where study_activity_id = :activityId))")
+    @SqlUpdate("insert into pdf_answer_substitution (pdf_substitution_id, question_stable_code_id, parent_question_stable_code_id) "
+            + "(select :subId, "
+            + "(select question_stable_code_id from question_stable_code where stable_id = :stableId "
+            + "    and umbrella_study_id = (select study_id from study_activity where study_activity_id = :activityId)) "
+            + " , "
+            + "(select question_stable_code_id from question_stable_code where stable_id = :parentStableId "
+            + "    and umbrella_study_id = (select study_id from study_activity where study_activity_id = :activityId)) "
+            + "   from dual) ")
```

Comment:
How about cleaning up the statement to use regular `insert .. values (:subId, (select ..), (select ..))` instead of selecting from `dual`?